geospaas_harvesting/ingesters.py

```diff
@@ -446,7 +446,6 @@ def _get_normalized_attributes(self, url, *args, **kwargs):
         normalized_attributes['geospaas_service_name'] = 'ftp'
         normalized_attributes['geospaas_service'] = 'ftp'
         #Temporary solution
-        normalized_attributes['entry_id'] = urlparse(url).path.split('/')[-1]#'entry_id'is file name
```

Comment:
please also remove the comment

Reply:
Done in 50b577a. Thanks! I have forgotten that one!